clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/KubernetesCloudProvider.java

```diff
@@ -1,17 +1,18 @@
 /*
- * Copyright 2015 Google, Inc.
+ * Copyright 2020 Google, Inc.
```

Comment:
We don't normally update license headers on existing files. Was this on purpose?

Reply:
Thanks for catching this; all fixed!